upup/models/cloudup/resources/addons/networking.projectcalico.org.canal/k8s-1.16.yaml.template

```diff
@@ -631,6 +642,12 @@ spec:
       labels:
         k8s-app: canal
         role.kubernetes.io/networking: "1"
+      annotations:
+        # This, along with the CriticalAddonsOnly toleration below,
+        # marks the pod as a critical add-on, ensuring it gets
+        # priority scheduling and that its resources are reserved
+        # if it ever gets evicted.
+        scheduler.alpha.kubernetes.io/critical-pod: ''
```

Comment:
Same here, may be worth submitting a PR to Calico to remove it from their templates.

Reply:
True. Removed.

Reply:
Probably a good idea to submit a PR there. There are a few more cosmetic things that should be fixed.